apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.tsx

```diff
@@ -16,18 +16,28 @@ export const ButtonVariantTest: React.FunctionComponent = () => {
   const [showFABText, setShowFABText] = React.useState(true);
   const flipFABcontent = React.useCallback(() => setShowFABText(!showFABText), [showFABText]);
 
+  const isMobile = ['android', 'ios'].includes(Platform.OS as string);
+  const platfromSpecificNaming = isMobile ? 'Accent' : 'Primary';
+
   return (
     <View style={testContentRootViewStyle}>
       <Button style={commonTestStyles.vmargin}>Default</Button>
       <Button disabled style={commonTestStyles.vmargin}>
         Default Disabled
       </Button>
       <Button appearance="primary" style={commonTestStyles.vmargin}>
-        Primary
+        {platfromSpecificNaming}
       </Button>
       <Button disabled appearance="primary" style={commonTestStyles.vmargin}>
-        Primary Disabled
+        {`${platfromSpecificNaming} Disabled`}
       </Button>
+      {isMobile && (
+        <>
+          <Button appearance="outline" style={commonTestStyles.vmargin}>
+            Outline
+          </Button>
+        </>
+      )}
       <Button appearance="subtle" style={commonTestStyles.vmargin}>
         Subtle
       </Button>
```

Comment:
The components not there is fluent guidelines for android should not be rendered atleast for tester. IMO this is confusing for usage - my opinion on such code would be to not show in tester for an endpoint, as well as mentioned in SPEC. My thought is it's not an approved Fluent component in android.